pkg/config/stepmeta.go

```diff
@@ -374,6 +363,19 @@ func (m *StepData) GetResourceParameters(path, name string) map[string]interface
 	return resourceParams
 }
 
+func (container *Container) commonConfiguration(keyPrefix string, config *map[string]interface{}) {
+	putMapIfNotEmpty(*config, keyPrefix+"EnvVars", EnvVarsAsMap(container.EnvVars))
+	putStringIfNotEmpty(*config, keyPrefix+"Image", container.Image)
+	putStringIfNotEmpty(*config, keyPrefix+"Name", container.Name)
+	if container.ImagePullPolicy != "" {
+		(*config)[keyPrefix+"PullImage"] = container.ImagePullPolicy != "Never"
+	}
+	putStringIfNotEmpty(*config, keyPrefix+"Workspace", container.WorkingDir)
+	putSliceIfNotEmpty(*config, keyPrefix+"Options", OptionsAsStringSlice(container.Options))
+	//putSliceIfNotEmpty(*config, keyPrefix+"VolumeBind", volumeMountsAsStringSlice(container.VolumeMounts))
+
+}
```

Comment:
@marcusholl @stippi2 Here I export the common configuration of main and sidecar container. What do you think?

Reply:
I am always in favor of eliminating code duplication. That I forgot to add the changes in the second copy makes it perfectly clear why.